card.html

```diff
@@ -0,0 +1,135 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="utf-8">
+	<title>Animal Trading Cards</title>
+	<link rel="stylesheet" type="text/css" href="./styles.css" />
+</head>
+<body>
+	<!-- outer-container for flexbox -->
+	<div class="outer-container">
+
+	  <div class="card blackbear">
+		<h3 class="card-name">American Black Bear</h3>
+		<img src="blackbear.jpg" alt="black bear walking in field">
+		<div class="card-text">
+			<p class="quick-fact">There are nearly one million black bears in North America!</p>
+			<ul>
+				<li><span class="fact-labels">Scientific Name</span>: Ursus americanus</li>
+				<li><span class="fact-labels">Weight Range</span>: 125-600lbs</li>
+				<li><span class="fact-labels">Habitat</span>: Forests, wooded mountains and swamps across the United States, Canada and even parts of Mexico.</li>
+				<li><span class="fact-labels">Typical Diet</span>:  Omnivore, but 95% plant-based—including grasses, leaves, seeds, nuts, fruits, berries, fish and insects. </li>
```

Comment:
I see your trying to add labels to this data 👍  - maybe investigate `aria-labelledby` or `dt/dl` https://www.w3schools.com/tags/tag_dl.asp. Could make it more semantic.